webapp/channels/src/components/suggestion/suggestion_box/suggestion_box.jsx

```diff
@@ -564,7 +564,38 @@ export default class SuggestionBox extends React.PureComponent {
         return this.state.items.some((item) => !item.loading);
     };
 
+    confirmPretext = () => {
+        const textbox = this.getTextbox();
+        const pretext = textbox.value.substring(0, textbox.selectionEnd).toLowerCase();
+
+        if (this.pretext !== pretext) {
```

Comment:
I'm not entirely sure, but I wonder if we still need to check `this.composing` here as well. I don't know how using the arrow keys when typing a CJK character behaves or if it ends composition immediately

Reply:
I just tested it, and it doesn't end the composition. It just move around the composition. So... not sure either how you want to handle this. I didn't see any obvious failure, but 0/5.

Reply:
I'm not sure how it should work either. I thought we used to support autocompleting partially-composed Korean words like `~모ㅅ` would autocomplete to `~모사`, but at least on master, that doesn't appear to be the case. It's been ages since I've tried to use that though, so maybe it never worked.

As long as it works the same as it does currently when typing normally, that should be fine